<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterExtrationDialog</class>
 <widget class="QDialog" name="WaterExtrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water Extract</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input Image</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="7">
    <widget class="QLineEdit" name="leInputImage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8" colspan="2">
    <widget class="QPushButton" name="btnInputImage">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sub Dataset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cboSubDataset">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>AOI Image</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="7">
    <widget class="QLineEdit" name="leAoiImage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="2">
    <widget class="QPushButton" name="btnAoiImage">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Output Dir</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="7">
    <widget class="QLineEdit" name="leOutputDir">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="8" colspan="2">
    <widget class="QPushButton" name="btnOutputDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>EXP Threshold</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leExpThreshold">
     <property name="text">
      <string>-1000</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Low Range</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="leLowRange">
     <property name="text">
      <string>-3001</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>High Range</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7" colspan="3">
    <widget class="QLineEdit" name="leHighRange">
     <property name="text">
      <string>10001</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Histogram bins</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leHistogramBins">
     <property name="text">
      <string>500</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Radius bins</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="leRadiusBins">
     <property name="text">
      <string>13</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Iteration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7" colspan="3">
    <widget class="QLineEdit" name="leIteration">
     <property name="text">
      <string>5000</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="10">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input Band</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QComboBox" name="cboInputBand"/>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Input Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7" colspan="3">
    <widget class="QComboBox" name="cboInputType">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WaterExtrationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WaterExtrationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
